api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/dashboards_controller.rb

```diff
@@ -95,6 +95,7 @@ def dashboard_params
         whitelisted[:settings] = params[:dashboard][:metadata] || {}
       end
       .except(:metadata)
+      .merge(owner_type: "User", owner_id: current_user.id)
```

Comment:
but  the current code was using UserDashboard, and therefore was already creating a dashboard for which the owner was a user.

There is no backward issue.